app/controllers/posts_controller.rb

```diff
@@ -25,9 +25,19 @@ def edit
   # POST /posts
   def create
     @post = Post.new(post_params)
-
     if @post.save
-      redirect_to @post, notice: 'Post was successfully created.'
+      tag_params.split(",").each do |tag_new|
+        allready_exist = false
+        Tag.find_each do |tag_db|
```

Comment:
PO to był "break", żeby nie przechodzić całej tabeli. To jak przechodzić? 
  ----- Original Message ----- 
  From: Anna Ślimak 
  To: kraryk/test_blog 
  Cc: kraryk 
  Sent: Tuesday, January 26, 2016 3:37 PM
  Subject: Re: [test_blog] BŁąd przy wywoływaniu @post.tags (#3)


  In app/controllers/posts_controller.rb:

>      if @post.save
> -      redirect_to @post, notice: 'Post was successfully created.'
> +      tag_params.split(",").each do |tag_new|
> +        allready_exist = false
> +        Tag.find_each do |tag_db|
nie przechodz po calej tabeli! moze byc duzo danych w tabeli! zly nawyk

  —
  Reply to this email directly or view it on GitHub.